playground/orleans/OrleansServiceDefaults/Extensions.cs

```diff
@@ -42,8 +42,10 @@ public static IHostApplicationBuilder ConfigureOpenTelemetry(this IHostApplicati
         builder.Services.AddOpenTelemetry()
             .WithMetrics(metrics =>
             {
-                metrics.AddRuntimeInstrumentation()
-                       .AddBuiltInMeters();
+                metrics.AddAspNetCoreInstrumentation()
+                       .AddHttpClientInstrumentation()
+                       .AddRuntimeInstrumentation()
+                       .AddMeter("Microsoft.Orleans");
```

Comment:
I figured it was nicer to hoist this out for just the one meter, but I can put it back as it was if that's preferred (unless this has its own extension method now too).